src/components/lyne-toast/lyne-toast.custom.d.ts

```diff
@@ -0,0 +1,75 @@
+import { AnimationBuilder } from '../../global/core/components/animations/animation-interface';
+
+export interface InterfaceLyneToastAttributes {
+  verticalPosition: 'top' | 'bottom';
+  horizontalPosition: 'left' | 'center' | 'right' | 'start' | 'end';
+  politeness: 'off' | 'assertive' | 'polite';
+}
+
+type InterfaceToastType = 'link' | 'action' | 'icon';
```

Comment:
should that no also be exported?

Reply:
In my opinion, InterfaceToastCommonAction is extended by InterfaceToast (Link / Action / Icon) so it's not useful on its own for consumers. 
The same for InterfaceToastType; it only serves to define the type property in InterfaceToast (Link / Action / Icon).

But on the other hand, I don't really know how the compilation happens: without 'export', are the types defined anyway (ie, does the consumer see the correct types)? if not, I add the export..

Reply:
@kyubisation Can you maybe answer as you know more in this topic?